### PR TITLE
Add superscript subscript

### PR DESCRIPTION
**Pull request by IlyaKozlov**

### CODE REVIEW

File: dedoc/data_structures/concrete_annotations/subscript_annotation.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+class SubscriptAnnotation(Annotation):
+
+    name = "subscript"
+    valid_values = ["True", "False"]
+
+    def __init__(self, start: int, end: int, value: str):
+        try:
+            bool(value)
+        except ValueError:
+            raise ValueError("the value of bold annotation should be True or False")
```
Comment (NastyBoget) [inline dedoc/data_structures/concrete_annotations/subscript_annotation.py:15]: не bold, a subscript

File: dedoc/data_structures/concrete_annotations/superscript_annotation.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+class SuperscriptAnnotation(Annotation):
+
+    name = "superscript"
+    valid_values = ["True", "False"]
+
+    def __init__(self, start: int, end: int, value: str):
+        try:
+            bool(value)
+        except ValueError:
+            raise ValueError("the value of bold annotation should be True or False")
```
Comment (NastyBoget) [inline dedoc/data_structures/concrete_annotations/superscript_annotation.py:15]: не bold, a superscript

Comment (IlyaKozlov) [reply dedoc/data_structures/concrete_annotations/superscript_annotation.py:15]: done

File: dedoc/data_structures/concrete_annotations/subscript_annotation.py (hunk 1 above)
Comment (IlyaKozlov) [reply dedoc/data_structures/concrete_annotations/subscript_annotation.py:15]: done

**Pull request merged**